build.gradle

```diff
@@ -16,37 +16,39 @@
  */
 
 plugins {
-    id 'org.jetbrains.intellij' version '1.16.1'
+    id 'org.jetbrains.intellij'
+    id 'java'
+    id 'checkstyle'
+    id 'com.github.spotbugs'
 }
 
-version = ballerinaPluginVersion
-
-repositories {
+ repositories {
+    gradlePluginPortal()
     mavenCentral()
     maven { url 'https://jitpack.io' }
 }
```

Comment:
Better to manage these in the `settings.gradle` file.

Reply:
@ThisaruGuruge Actually thats what I tried first, but it ended up with different errors while resolving dependencies from the mention repos. So this was temporarily added until we figure out the root cause. So I'll remove these entries with the fix